CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Support `OTEL_EXPORTER_ZIPKIN_ENDPOINT` env to specify zipkin collector endpoint (#2490)
 - Log the configuration of TracerProviders, and Tracers for debugging. To enable use a logger with Verbosity (V level) >=1
+- Added environment variables for: `OTEL_BSP_SCHEDULE_DELAY`, `OTEL_BSP_EXPORT_TIMEOUT`, `OTEL_BSP_MAX_QUEUE_SIZE` and `OTEL_BSP_MAX_EXPORT_BATCH_SIZE` (#2515)
```

Comment:
I think that we should document that we support these env vars. At least as a comment in `NewBatchSpanProcessor` so that it appears in GoDoc. Ideally also in some markdown file.

Reply:
These will show up in the constants of the GoDoc.  There isn't a readme for the SDK tracer, nor any of the components covered by the SDK.  So, I think it would be a large ask for that here.  If the GoDocs don't cover it let's start an issue to document the tracer in a more holistic manner.

Reply:
> These will show up in the constants of the GoDoc

that is good enough. Have not spotted that these env vars are exported.